generator.yml

```diff
@@ -297,6 +297,135 @@ builds:
+  # - {<< : *ppc32,         << : *mainline,         << : *llvm,            boot: true,  llvm_version: *llvm_11}
+  # - {<< : *ppc64,         << : *mainline,         << : *ppc64_llvm,      boot: true,  llvm_version: *llvm_11}
+  - {<< : *ppc64le,       << : *mainline,         << : *llvm,            boot: true,  llvm_version: *llvm_11}
+  - {<< : *riscv,         << : *mainline,         << : *riscv_llvm_full, boot: true,  llvm_version: *llvm_11}
+  - {<< : *s390,          << : *mainline,         << : *clang,           boot: true,  llvm_version: *llvm_11}
+  - {<< : *x86_64,        << : *mainline,         << : *llvm_full,       boot: true,  llvm_version: *llvm_11}
+  - {<< : *x86_64_allmod, << : *mainline,         << : *llvm_full,       boot: false, llvm_version: *llvm_11}
+  - {<< : *x86_64_allno,  << : *mainline,         << : *llvm_full,       boot: false, llvm_version: *llvm_11}
+  - {<< : *x86_64_allyes, << : *mainline,         << : *llvm_full,       boot: false, llvm_version: *llvm_11}
+  ##########
+  #  Next  #
+  ##########
+  - {<< : *arm32_v5,      << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_11}
+  - {<< : *arm32_v6,      << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_11}
+  - {<< : *arm32_v7,      << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_11}
+  - {<< : *arm32_v7_t,    << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_11}
+  - {<< : *arm32_allmod,  << : *next,             << : *llvm,            boot: false, llvm_version: *llvm_11}
+  - {<< : *arm32_allno,   << : *next,             << : *llvm,            boot: false, llvm_version: *llvm_11}
+  - {<< : *arm32_allyes,  << : *next,             << : *llvm,            boot: false, llvm_version: *llvm_11}
+  - {<< : *arm64,         << : *next,             << : *llvm_full,       boot: true,  llvm_version: *llvm_11}
+  - {<< : *arm64_allmod,  << : *next,             << : *llvm_full,       boot: false, llvm_version: *llvm_11}
+  - {<< : *arm64_allno,   << : *next,             << : *llvm_full,       boot: false, llvm_version: *llvm_11}
+  - {<< : *arm64_allyes,  << : *next,             << : *llvm_full,       boot: false, llvm_version: *llvm_11}
+  - {<< : *i386,          << : *next,             << : *llvm_full,       boot: true,  llvm_version: *llvm_11}
+  - {<< : *mips,          << : *next,             << : *mips_llvm,       boot: true,  llvm_version: *llvm_11}
+  - {<< : *mipsel,        << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_11}
+  # ppc32 and ppc64: Build disabled (https://github.com/ClangBuiltLinux/continuous-integration2/issues/2)
+  # - {<< : *ppc32,         << : *next,             << : *llvm,            boot: true,  llvm_version: *llvm_11}
+  # - {<< : *ppc64,         << : *next,             << : *ppc64_llvm,      boot: true,  llvm_version: *llvm_11}
```

Comment:
the can get turned on now, right?

Reply:
No, LLVM 11 does not have that bug fixed.

Reply:
Is https://github.com/ClangBuiltLinux/linux/issues/1244 the issue that should be linked in the comment, then?

Reply:
I can update it if you would like. That issue is linked in the Linaro issue that is linked in the continuous-integration issue.

Reply:
Even linking to the linaro issue would be one less level of indirection.  Sounds like tuxmake needs to permit setting  CROSS_COMPILE?

Reply:
> Even linking to the linaro issue would be one less level of indirection.

Sure, done.

> Sounds like tuxmake needs to permit setting CROSS_COMPILE?

Yes or going to a more neutral triple like `powerpc-linux-gnu-`.